e2e/test/scenarios/dashboard/dashboard-questions.cy.spec.js

```diff
@@ -33,8 +33,9 @@ describe("Dashboard > Dashboard Questions", () => {
       H.modal().findByText("Orders in a dashboard");
       H.modal().button("Save").click();
 
-      // should take you to the edit dashboard screen
+      // should take you to the edit dashboard screen + url has hash param to auto-scroll
       cy.url().should("include", "/dashboard/");
+      cy.location("hash").should("match", /scrollTo=\d+/);
```

Comment:
nit: you could write "url should have hash param to auto-scroll" as a comment above this line to make this clearer and more consistent with the other comments